lib/atecc508a/transport/i2c.ex

```diff
@@ -60,6 +60,22 @@ defmodule ATECC508A.Transport.I2C do
     rc
   end
 
+  @doc """
+  Detects if an ATECC508A is available at the address
+  """
+  @impl Transport
+  @spec detected?(instance()) :: boolean()
+  def detected?({i2c, address}) do
+    case wakeup(i2c, address) do
```

Comment:
Shoot. This is what I get for not merging the PR #17 that serializes the transport calls. Your PR looks good aside from it not using #17. I'm still testing that out. I'll resolve conflicts and merge this one afterwards.